velox/functions/lib/Re2Functions.cpp

```diff
@@ -470,6 +479,8 @@ class OptimizedLikeWithMemcmp final : public VectorFunction {
   vector_size_t reducedPatternLength_;
 };
 
+// This function is used when pattern and escape are constants. And there is not
+// fast path that avoids compiling the regular expression.
```

Comment:
`// And there is not fast path that avoids compiling the regular expression.
`Is this comment correct? It reads a bit strange.

Reply:
yeh like if there is a fast path and no regex need to be compiled we create  another function like             OptimizedLikeWithMemcmp<PatternKind::kExactlyN>>